### PR TITLE
[Outlook] (requirement set) Add Mailbox 1.12

### PR DESCRIPTION
**Pull request by samantharamon**
Related PRs: 
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61366
- https://github.com/OfficeDev/office-js-docs-pr/pull/3561
- https://github.com/OfficeDev/Office-Add-in-samples/pull/360

### CODE REVIEW

**Review by lindalu-MSFT — COMMENTED** [review]
Wow, 171 files! You have been super busy! As this is reference, I did not review all 171 files. I did open the first 25 and spot checked to be sure nothing was majorly broken and that links were working. If you need more in-depth than that, I'll need to schedule time to do it! I think you probably know better than I if it's ready to go or not!

File: docs/requirement-sets/outlook/outlook-api-requirement-sets.md [hunk 1]
```diff
@@ -129,11 +128,11 @@ Add-ins are supported in Outlook on the following platforms.
...
+> <sup>1</sup> Support for **1.8** in Outlook on Windows with a Microsoft 365 subscription or a retail one-time purchase is available from version 1910 (build 12130.20272). Support for **1.9** in Outlook on Windows with a Microsoft 365 subscription is available from version 2008 (build 13127.20296). Support for **1.10** in Outlook on Windows with a Microsoft 365 subscription is available from version 2104 (build 13929.20296). Support for **1.11** in Outlook on Windows with a Microsoft 365 subscription is available from version 2110 (build 14527.20226). Support for **1.12** in Outlook on Windows with a Microsoft 365 subscription is available from version 2206 (build 15330.20196). For more details according to your version, see the update history page for [Office 2021](/officeupdates/update-history-office-2021) or [Microsoft 365](/officeupdates/update-history-office365-proplus-by-date) and how to [find your Office client version and update channel](https://support.microsoft.com/office/932788b8-a3ce-44bf-bb09-e334518b8b19).
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/outlook-api-requirement-sets.md:135]: I'm still working on the version terminology so I'll take care of these instances when I finalize that but this should do for now.

Also, verify how this renders on the site if you accept this suggestion
```suggestion
> <sup>1</sup> Version support for more recent requirement sets in Outlook on Windows with a Microsoft 365 subscription or a retail one-time purchase as follows:
>
> - Support for **1.8** is available from version 1910 (build 12130.20272).
> - Support for **1.9** is available from version 2008 (build 13127.20296).
> - Support for **1.10** is available from version 2104 (build 13929.20296).
> - Support for **1.11** is available from version 2110 (build 14527.20226).
> - Support for **1.12** is available from version 2206 (build 15330.20196).
>
> For more details according to your version, see the update history page for [Office 2021](/officeupdates/update-history-office-2021) or [Microsoft 365](/officeupdates/update-history-office365-proplus-by-date) and how to [find your Office client version and update channel](https://support.microsoft.com/office/932788b8-a3ce-44bf-bb09-e334518b8b19).
```

File: docs/requirement-sets/outlook/requirement-set-1.12/office.context.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+<br>
+
+---
+---
+
+#### displayLanguage: String
+
+Gets the locale (language) in RFC 1766 Language tag format specified by the user for the UI of the Office client application.
+
+The `displayLanguage` value reflects the current **Display Language** setting specified with **File > Options > Language** in the Office client application.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.12/office.context.md:150]: May also need to update other instances
```suggestion
The `displayLanguage` value reflects the current **Display Language** setting specified with **File** > **Options** > **Language** in the Office client application.
```

File: generate-docs/GenerateDocs.sh [hunk 3]
```diff
@@ -103,6 +105,7 @@ node whats-new excel ../api-extractor-inputs-excel-release/Excel_1_2/excel.d.ts
 node whats-new excel ../api-extractor-inputs-excel-release/Excel_1_1/excel.d.ts ./tool-inputs/excel-base.d.ts ../../docs/includes/excel-1_1
 
 node whats-new outlook ../api-extractor-inputs-outlook/outlook.d.ts ../api-extractor-inputs-outlook-release/outlook_1_11/outlook.d.ts ../../docs/includes/outlook-preview
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/GenerateDocs.sh:107]:
```suggestion
node whats-new outlook ../api-extractor-inputs-outlook/outlook.d.ts ../api-extractor-inputs-outlook-release/outlook_1_12/outlook.d.ts ../../docs/includes/outlook-preview
```

**Comment by samantharamon** [comment]
> Wow, 171 files! You have been super busy! As this is reference, I did not review all 171 files. I did open the first 25 and spot checked to be sure nothing was majorly broken and that links were working. If you need more in-depth than that, I'll need to schedule time to do it! I think you probably know better than I if it's ready to go or not!

Thanks, @lindalu-MSFT! Apologies, I should have specified which particular doc requires an edit pass, as most of the files are part of the build. If you get a chance, could you review [Outlook add-in API requirement set 1.12](https://review.docs.microsoft.com/en-us/javascript/api/requirement-sets/outlook/requirement-set-1.12/outlook-requirement-set-1.12?view=word-js-preview&branch=pr-en-us-1387) (file: [docs/requirement-sets/outlook/requirement-set-1.12/outlook-requirement-set-1.12.md](https://github.com/OfficeDev/office-js-docs-reference/blob/samramon-mailbox-1.12/docs/requirement-sets/outlook/requirement-set-1.12/outlook-requirement-set-1.12.md))?

File: docs/requirement-sets/outlook/requirement-set-1.12/office.context.md (hunk 2 above)
Comment (samantharamon) [reply docs/requirement-sets/outlook/requirement-set-1.12/office.context.md:150]: Thanks for the suggested change. I'll update this for other applicable requirement sets in a separate PR.

**Comment by lindalu-MSFT** [comment]
Only comment is under "What's new in 1.12, there are three bullet points. One and Three have a link but two does not. Is that intentional or is a link missing in the 2nd bullet?

File: docs/docs-ref-autogen/outlook_1_12/outlook/office.appointmentcompose.yml [hunk 4]
```diff
@@ -0,0 +1,3511 @@
...
+uid: 'outlook!Office.AppointmentCompose:interface'
+package: outlook!
+fullName: Office.AppointmentCompose
+summary: >-
+  The appointment organizer mode of [Office.context.mailbox.item](xref:outlook!Office.Item:interface)<!-- -->.
+
+
+  **Important**: This is an internal Outlook object, not directly exposed through existing interfaces. You should treat
+  this as a mode of `Office.context.mailbox.item`<!-- -->. For more information, refer to the [Object
+  Model](https://docs.microsoft.com/javascript/api/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item)
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/docs-ref-autogen/outlook_1_12/outlook/office.appointmentcompose.yml:12]: Thanks for catching that. I'll address this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61366 and update our process.

File: docs/docs-ref-autogen/outlook_1_12/outlook/office.messageread.yml [hunk 5]
```diff
@@ -0,0 +1,3166 @@
...
+  - name: 'addHandlerAsync(eventType, handler, options, callback)'
+    uid: 'outlook!Office.MessageRead#addHandlerAsync:member(1)'
+    package: outlook!
+    fullName: 'addHandlerAsync(eventType, handler, options, callback)'
+    summary: >-
+      Adds an event handler for a supported event. **Note**: Events are only available with task pane implementation.
+
+
+      For supported events, refer to the Item object model [events
+      section](https://docs.microsoft.com/javascript/api/requirement-sets/outlook/requirement-set-1.11/office.context.mailbox.item#events)<!--
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/docs-ref-autogen/outlook_1_12/outlook/office.messageread.yml:1206]: Thanks. I'll address this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61366.

**Comment by samantharamon** [comment]
> Only comment is under "What's new in 1.12, there are three bullet points. One and Three have a link but two does not. Is that intentional or is a link missing in the 2nd bullet?

@lindalu-MSFT It was intentional, but now that I've reviewed it again based on your comment, I've added a link to the send options page.

File: docs/requirement-sets/requirement-sets.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office client application and platform availability for Office Add-ins
 description: Supported requirement sets for Excel, OneNote, Outlook, PowerPoint, Project, and Word.
-ms.date: 06/29/2022
+ms.date: 07/20/2022
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/requirement-sets.md:4]:
```suggestion
ms.date: 09/09/2022
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**